python/ray/autoscaler/_private/command_runner.py

```diff
@@ -737,31 +737,23 @@ def _check_if_container_restart_is_needed(
         try:
             active_mounts = json.loads(mounts)
             active_remote_mounts = {
-                mnt["Destination"]
+                mnt["Destination"].strip("/")
                 for mnt in active_mounts
             }
             # Ignore ray bootstrap files.
             requested_remote_mounts = {
-                self._docker_expand_user(remote)
+                self._docker_expand_user(remote).strip("/")
```

Comment:
the output of the docker response actually automatically strips the trailing "/", so this previously would show *all* file mounts listed in the yaml.

Reply:
Thanks for catching this!!